dynawo/3rdParty/libiidm/xml/include/IIDM/xml/ExecUtils.h

```diff
@@ -32,3 +34,5 @@
 std::string getEnvVar(std::string const& key);
 
 #endif  // LIBIIDM_XML_INTERNALS_IMPORT_GUARD_EXECUTILS_H
+
+} // end of namespace IIDM::
```

Comment:
extra ::

Reply:
I kept the same coding style present in the lib but I can change it

Reply:
Not needed then